scripts/make/build-frontend.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+cd ../..
+repo_root=$(pwd)
+
+source "$repo_root/.build.env"
+
+if [ "$frontend_build" == "Y" ]; then
+  cd "$frontend_dir" && "$frontend_pipelines_command"
```

Comment:
Looks like `frontend_pipelines_command` fetches dependencies and runs a build task? I think this one needs splitting into two - `install-frontend.sh` and `build-frontend.sh`? Or just put the commands directly in the makefile:

```
install-frontend:
  yarn

build-frontend:
  yarn build
```

Reply:
I guess we should do this if we're being rigourous....

Reply:
This script is only ever called by Bitbucket Pipelines. We could separate the files as you suggest but it simply means that rather than just calling the Makefile target `build-frontend` in the "Frontend" step of our stock Bitbucket Pipelines config you would have to call two Makefile targets such as `install-frontend` and `build-frontend`.

This is probably a good one for the team to vote on.